tests/lora/test_lora_layers_flux2.py

```diff
@@ -94,6 +95,44 @@ def get_dummy_inputs(self, with_generator=True):
 
         return noise, input_ids, pipeline_inputs
 
+    def test_lora_fuse_nan(self):
```

Comment:
```suggestion
    # Overriding because (1) text encoder LoRAs are not supported in Flux 2 and (2) because the Flux 2 single block
    # QKV projections are always fused, it has no `to_q` param as expected by the original test.
    def test_lora_fuse_nan(self):
```
I think adding a comment explaining why `test_lora_fuse_nan` is being overridden would be useful. The suggestion contains my understanding of the reasons for overriding.